src/Parser/Mage/Fire/Modules/Features/Combustion.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatNumber } from 'common/format';
+import Analyzer from 'Parser/Core/Analyzer';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+
+class Combustion extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    spellUsable: SpellUsable,
+  };
+
+  lowPhoenixFlamesCharges = 0;
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.COMBUSTION.id) {
+      return;
+    }
+    const phoenixFlamesCharges = this.spellUsable.chargesAvailable(SPELLS.PHOENIXS_FLAMES.id);
+    if (phoenixFlamesCharges < 2) {
+      this.lowPhoenixFlamesCharges += 1;
+    }
+  }
+
+  get phoenixFlamesChargesSuggestionThresholds() {
```

Comment:
Since you are already in the Combustion module, you could shorten then to `suggestionThresholds()` to just clean up the below usage of it.